### ISSUE TITLE
Training ESRGAN on PROBA-V

### ISSUE THREAD

**Issue by yunseok624** [allenai/satlas-super-resolution#13]
Hello!

I'm trying to train the esrgan on proba-v dataset and I came across some questions I coudln't find in the paper.
1. Did you pre-train a PSNR-oriented model with the L1 loss with proba-v dataset (just like in esrgan and real-esrgan did)?
2. In the probav_esrgan.yml I don't understand the number of input channels. in RRDBNet shouldn't the input channel be 9 since proba-v (from the competition) has only 1 channel? I also do not understand why the input channel for discriminator is 30.
![image](https://github.com/allenai/satlas-super-resolution/assets/58263915/3151841a-a415-49ba-997e-7bac87402b1d)

Thank you in advance.

**Comment by piperwolters** [comment]
Hello, the experiments branch is a work in progress. You can adjust the args in the config to match what you have described. And the discriminator takes in the low-res images + the real/fake image as input.

We did not pretrain a PSNR-oriented model like in real-esrgan. We just trained the whole model with all losses that are specified in the config files.

And yes, we used imagery from the PROBA-V challenge.
